invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -70,7 +70,7 @@ class Meta:
     is_published = fields.Boolean(dump_only=True)
     status = fields.String(dump_only=True)
 
-    # stats = NestedAttribute(StatsSchema, dump_only=True)
+    stats = fields.Dict(attribute="stats", dump_only=True)
```

Comment:
What happens if I create a record, and I pass a `stats` field? It should be ignored/removed because of the `dump_only` and also because of the `pass` in the `load` method.

Reply:
yes, exactly.
if a `stats` field is provided, it will be ignored.